pages/task/content/status.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     hint: {
       application: 'The application will be sent to an inspector for assessment.',
       amendment: 'The amendment will be sent to an inspector for assessment.',
-      revocation: 'The revocation will be sent to an inspector for assessment.'
+      revocation: 'The revocation request will be sent to an inspector for assessment.'
```

Comment:
Is that true?

Reply:
Sorry, my bad, I was missing the context.

I thought this was on an initial submission of the request, but it isn't.